exercises/practice/luhn/test-words.8th

```diff
@@ -1,10 +1,21 @@
 needs console/loaded
 
+var test-count
```

Comment:
```suggestion
-1 var, test-count
```
Just `var test-count` initializes the variable to zero. If the student's solution is just `bye` then they _won't_ see the "not all tests completed" error message because the program will exit before loading the exercise tests (with the `!test-count` call).

Reply:
Hmmm ... hadn't thought about that.